app/helpers.php

```diff
@@ -89,13 +97,13 @@ function ($language) {
         unset($result['en']);
         $fr = $result['fr'];
         unset($result['fr']);
-        $ase = $result['ase'];
-        unset($result['ase']);
-        $fcs = $result['fcs'];
-        unset($result['fcs']);
+        $asl = $result['asl'];
+        unset($result['asl']);
+        $lsq = $result['lsq'];
+        unset($result['lsq']);
 
         if ($signed) {
-            foreach (['fcs', 'ase', 'fr', 'en'] as $code) {
+            foreach (['lsq', 'asl', 'fr', 'en'] as $code) {
```

Comment:
Consider sourcing the locales from config. In this case you might want to provide a sort order. However, in this case just reversing the array is probably enough.

```suggestion
            foreach (array_reverse(config('locales.supported')) as $code) {
```